rc/filetype/elixir.kak

```diff
@@ -92,12 +92,25 @@ define-command -hidden elixir-trim-indent %{
     try %{ execute-keys -draft -itersel <a-x> s \h+$ <ret> d }
 }
 
-define-command -hidden elixir-insert-on-new-line %{
-    evaluate-commands -draft -itersel %{
-        # copy -- comments prefix and following white spaces
-        try %{ execute-keys -draft k <a-x> s ^\h*\K--\h* <ret> y gh j P }
-    }
-}
+define-command -hidden elixir-insert-on-new-line %[
+    evaluate-commands -no-hooks -draft -itersel %[
+        # copy '#' comment prefix and following white spaces
+        try %{ exec -draft k <a-x> s ^\h*\K#\h* <ret> y jgi P }
```

Comment:
```suggestion
        try %{ execute-keys -draft k <a-x> s ^\h*\K#\h* <ret> y jgi P }
```

for consistency with other rc files, same for the ruby PR

Reply:
Sure, done